code/modules/mapping/mapping_helpers.dm

```diff
@@ -101,3 +101,28 @@ GLOBAL_LIST_EMPTY(z_is_planet)
 	GLOB.z_is_planet["[T.z]"] = TRUE
 	return INITIALIZE_HINT_QDEL
 
+/obj/effect/mapping_helpers/template_spawner //spawning templates
+	name = "template spawner"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "syndballoon"
```

Comment:
It would be nice if while you were here you moved the `icon`/`icon_state`/`layer` duplication on all the subtypes of `mapping_helpers` onto `mapping_helpers` itself.